src/main/pg/rx.h

```diff
@@ -24,7 +24,7 @@
 
 #include "pg/pg.h"
 
-#define GET_FRAME_ERR_LPF_FREQUENCY(period) (1 / (period / 10.0f))
+#define GET_FRAME_ERR_LPF_FREQUENCY(period) (1.0f / ((period) * 0.1f/*[s]*/))
```

Comment:
I guess the intent is that the LPF cutoff is set to ten times the inverse of the period ie 10Hz when period is 1s.
Perhaps it might be easier to conceptualise that relationship with:
```
define GET_FRAME_ERR_LPF_FREQUENCY(period) (10.0f / period)    // period is in seconds
```

Reply:
https://github.com/betaflight/betaflight/blob/54570328388190eb48900e0db6f2f271192019a7/src/main/pg/rx.h#L59

it is in 0.1s units